src/components/OfferMyAccountCard/OfferMyAccountCard.tsx

```diff
@@ -42,6 +44,9 @@ import { OfferMyAccountCardProps } from './OfferMyAccountCard.types';
 const OfferMyAccountCard = ({ offerId }: OfferMyAccountCardProps) => {
   const { t } = useTranslation();
   const dispatch = useAppDispatch();
+
+  const [switches, setSwitches] = useState<SwitchDetail[]>([]);
```

Comment:
shouldn't this be in redux? seems like all other data that is similar to this is kept in redux 🤔

Reply:
This is only used in this one place so I don't see the point of throwing it into the global store, a little extra code will be needed and it may be a slight littering of the store. Only when this state will be needed elsewhere in the application would I add it to the global store.